html-css/lesson1.md

```diff
@@ -99,7 +100,7 @@ Not everyone uses a mouse. Some users will prefer a keyboard, where they can hit
 }
 ```
 
-> **Exercise:** Work in pairs and use the pseudo classes to make the background color of the red buttons change when in a "hover" or "focus" state. See if you can make the white "Volunteer" button change to a different background without effecting the red buttons.
+> **Exercise (pair programming):** Add the 'Learn more' section and make each box highlighted when in a "hover" or "focus" state.
```

Comment:
Ha! Also, the learn more section is already there - what they need to do is make it into columns.